verify.go

```diff
@@ -55,34 +55,6 @@ func onReady2(discord *discordgo.Session, ready *discordgo.Ready) {
 		if DB == nil {
 			return
 		}
-		for i := 1; i < 5; i++ {
-			for _, id := range staffIDs[i] {
-				var num int
-				err := DB.QueryRow("SELECT nick FROM nicks WHERE id = $1", id).Scan(&num)
-				if err != nil {
-					err = DB.QueryRow("INSERT INTO nicks(id) VALUES ($1) RETURNING (nick)", id).Scan(&num)
-					if err != nil {
-						panic(err)
-					}
-				}
-				meme(num, id)
-			}
-		}
-	}()
```

Comment:
You included too many brackets >.>
Line 70 should've been the last bracket to delete
Could you commit this fix to your branch?